src/API/provision.js

```diff
@@ -0,0 +1,6 @@
+import { rocketsAPI, missionsAPI } from './API';
+
+const getRockets = () => rocketsAPI.get();
+const getMissions = () => missionsAPI.get();
+
+export { getMissions, getRockets };
```

Comment:
We don't need a provision file, because we already set up a service so, to make our code clean and dry, kindly, remove the provision and add your code to services.